### PR TITLE
Add mentions of Cloud Foundry support, etc.

### PR DESCRIPTION
**Pull request by fifthposition**
Mostly updating various pages to include mentions of CF support. Also added short section to Halyard "Custom Configuration" reference page, to talk about adding settings for the Deck service.

cc @dorbin

### CODE REVIEW

File: reference/halyard/custom.md [hunk 1]
```diff
@@ -42,6 +42,18 @@ information.
 You can validate that your Custom Profiles have been picked up and applied to
 their services by checking `~/.hal/$DEPLOYMENT/history/service-profiles.yml`.
 
+### Custom Profile for Deck
+
+To supply custom settings or new values that override generated settings for the
+Deck service, you can place them in a file called `settings-local.js`. For
+example, to enable the `artifactsRewrite` feature flag, you can create the
```
Comment (dorbin) [inline reference/halyard/custom.md:49]: @fifthposition 
Is this flag going to live on, or is it just temporary?

File: reference/halyard/custom.md [hunk 2]
```diff
@@ -42,6 +42,18 @@ information.
 You can validate that your Custom Profiles have been picked up and applied to
 their services by checking `~/.hal/$DEPLOYMENT/history/service-profiles.yml`.
 
+### Custom Profile for Deck
+
+To supply custom settings or new values that override generated settings for the
+Deck service, you can place them in a file called `settings-local.js`. For
```
Comment (dorbin) [inline reference/halyard/custom.md]: s/service/UI/
(just to make it clear what Deck is, for newer users who might not know)

**Review by dorbin — COMMENTED** [review]
@fifthposition 
Just a couple of comments

File: reference/halyard/custom.md (hunk 1 above)
Comment (fifthposition) [reply reference/halyard/custom.md:49]: It's temporary (hopefully going away in 1.15, when hopefully these changes will be default).

Comment (dorbin) [reply reference/halyard/custom.md:49]: When that happens, will you or I remember to update this?

Comment (fifthposition) [reply reference/halyard/custom.md:49]: If we document the `artifactsRewrite` changes as being behind `artifactsRewrite`, then I'll remember. :)

**Review by dorbin — APPROVED** [review]

**Pull request merged**